daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -4,6 +4,9 @@
 package btrfs // import "github.com/docker/docker/daemon/graphdriver/btrfs"
 
 /*
+// keep struct field name compatible with btrfs-progs < 6.1.
+#define max_referenced max_rfer
+
 #include <stdlib.h>
 #include <dirent.h>
 #include <btrfs/ioctl.h>
```

Comment:
```suggestion
#include <stdlib.h>
#include <dirent.h>

// keep struct field name compatible with btrfs-progs < 6.1.
#define max_referenced max_rfer
#include <btrfs/ioctl.h>
```

Just to be safe, in case the libc headers contain a `max_referenced` token.

Reply:
done